<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор сетевых схем</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/rec/img/icon_application.png</normaloff>:/rec/img/icon_application.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Оборудование&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="widget_equipments"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPushButton" name="button_add_equipment">
                   <property name="text">
                    <string>Добавить объект</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="button_change_description">
                   <property name="text">
                    <string>Изменить описание</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QPushButton" name="button_add_connection">
                   <property name="text">
                    <string>Создать связь</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="button_delete_object">
                   <property name="text">
                    <string>Удалить объект</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTextEdit" name="descriprion"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Схема соединений&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGraphicsView" name="graphicsView"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="create_project"/>
    <addaction name="open_project"/>
    <addaction name="separator"/>
    <addaction name="save_project"/>
    <addaction name="separator"/>
    <addaction name="save_as_project"/>
    <addaction name="close_project"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Оборудование</string>
    </property>
    <addaction name="open_equipmentCreator"/>
    <addaction name="open_connectorCreator"/>
    <addaction name="open_library"/>
    <addaction name="add_rule"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="instruction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="create_project"/>
   <addaction name="open_project"/>
   <addaction name="save_project"/>
   <addaction name="save_as_project"/>
   <addaction name="open_equipmentCreator"/>
   <addaction name="open_connectorCreator"/>
   <addaction name="open_library"/>
   <addaction name="add_rule"/>
   <addaction name="instruction"/>
   <addaction name="close_project"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="create_project">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/create new.png</normaloff>:/rec/img/create new.png</iconset>
   </property>
   <property name="text">
    <string>Создать проект</string>
   </property>
  </action>
  <action name="open_project">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/open.png</normaloff>:/rec/img/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть проект</string>
   </property>
  </action>
  <action name="save_project">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/save.png</normaloff>:/rec/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="close_project">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/exit.png</normaloff>:/rec/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="open_equipmentCreator">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/equipment.png</normaloff>:/rec/img/equipment.png</iconset>
   </property>
   <property name="text">
    <string>Создать оборудование</string>
   </property>
  </action>
  <action name="open_library">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/add.png</normaloff>:/rec/img/add.png</iconset>
   </property>
   <property name="text">
    <string>Открыть библиотеку</string>
   </property>
  </action>
  <action name="save_as_project">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/save as.png</normaloff>:/rec/img/save as.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="add_rule">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/connection.png</normaloff>:/rec/img/connection.png</iconset>
   </property>
   <property name="text">
    <string>Добавить соединение</string>
   </property>
  </action>
  <action name="open_connectorCreator">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/connector.png</normaloff>:/rec/img/connector.png</iconset>
   </property>
   <property name="text">
    <string>Создать коннектор</string>
   </property>
  </action>
  <action name="instruction">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/img/help.png</normaloff>:/rec/img/help.png</iconset>
   </property>
   <property name="text">
    <string>Инструкция</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
